native/src/widget/image.rs

```diff
@@ -125,6 +125,19 @@ impl Handle {
         Self::from_data(Data::Path(path.into()))
     }
 
+    /// Creates an image [`Handle`] containing the image pixels directly.
+    ///
+    /// This is useful if you have already decoded your image.
+    ///
+    /// [`Handle`]: struct.Handle.html
```

Comment:
We should probably mention the BGRA format requirement here too.

Reply:
Done!